src/components/atoms/Input/index.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { FC } from "react";
+
+interface IInput {
+  type: string;
+  name: string;
+  value?: string | number;
+  ref?: React.Ref<HTMLInputElement>;
+  placeholder?: string;
+  onChange?: (event: React.ChangeEvent<HTMLInputElement>) => void;
+  className?: "filled" | "transparent";
+}
+
+const Input: FC<IInput> = ({
+  type,
+  name,
+  value,
+  ref,
+  placeholder,
+  onChange,
+  className,
+}) => {
+  const generateClassName = () => {
+    switch (className) {
```

Comment:
rename to variant 
as className should be separated prop in case there would be a need to move button or smth like this